tss/coordinator.go

```diff
@@ -69,10 +74,11 @@ func NewCoordinator(
 
 // Execute calculates process leader and coordinates party readiness and start the tss processes.
 func (c *Coordinator) Execute(ctx context.Context, tssProcess TssProcess, resultChn chan interface{}) error {
```

Comment:
Have some traces in this would be quite usefull.

Reply:
added